frontend/public/components/utils/details-page.tsx

```diff
@@ -157,6 +158,25 @@ export const ResourceSummary: React.FC<ResourceSummaryProps> = ({
       <DetailsItem label={t('public~Owner')} obj={resource} path="metadata.ownerReferences">
         <OwnerReferences resource={resource} />
       </DetailsItem>
+      <DetailsItem label={t('public~Upstream configuration')} obj={resource} path="spec.upstream">
```

Comment:
We might want to make this a separate component that's used here and on the cluster settings page.

Reply:
Done!